zk/stages/stage_l1syncer.go

```diff
@@ -227,6 +227,10 @@ func parseLogType(l1RollupId uint64, log *ethTypes.Log) (l1BatchInfo types.L1Bat
 		stateRoot, l1InfoRoot common.Hash
 	)
 
+	if l1RollupId != 1 && isRollupIdMatching == false {
```

Comment:
Maybe this is dangerous we also need to check that the data doesn't contain numBatches to ensure this is definitely a rollupid we compared on

Reply:
Correct if I'm wrong but currently parseLogType function always return "valid" objects.

With this modification it could return empty objects. Are we sure that all the places where parseLogType is invoked can handle this scenario?